### PR TITLE
Dockerfiles for backends and frontends and docker compose

### PR DESCRIPTION
**Pull request by kajedot**

### CODE REVIEW

File: frontend/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+USER node
+
+RUN npm ci
+
+COPY --chown=node:node . .
+
+EXPOSE 5173
+
+CMD ["npm", "run", "dev"]
```
Comment (LissaGreense) [inline frontend/Dockerfile:21]: We can't run dev server in docker file

File: frontend/package.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
   "version": "0.0.0",
   "type": "module",
   "scripts": {
-    "dev": "vite",
+    "dev": "vite --host",
```
Comment (LissaGreense) [inline frontend/package.json:7]: remove it

**Review by LissaGreense — CHANGES_REQUESTED** [review]

File: frontend/Dockerfile (hunk 1 above)
Comment (kajedot) [reply frontend/Dockerfile]: fixed: CMD ["npm", "run", "preview"]

File: frontend/package.json (hunk 2 above)
Comment (kajedot) [reply frontend/package.json:7]: I would leave it for testing

Comment (kajedot) [reply frontend/package.json]: deleted

**Review by LissaGreense — APPROVED** [review]
nice

**Pull request merged**